flink-connectors/flink-connector-rabbitmq/src/test/java/org/apache/flink/streaming/connectors/rabbitmq/RMQSinkTest.java

```diff
@@ -83,7 +103,22 @@ public void throwExceptionIfChannelIsNull() throws Exception {
 	}
 
 	private RMQSink<String> createRMQSink() throws Exception {
-		RMQSink rmqSink = new RMQSink<String>(rmqConnectionConfig, QUEUE_NAME, serializationSchema);
+		RMQSink<String> rmqSink = new RMQSink<String>(rmqConnectionConfig, QUEUE_NAME, serializationSchema);
+		rmqSink.open(new Configuration());
+		return rmqSink;
+	}
+
+	private RMQSink<String> createRMQSinkFeatured() throws Exception {
```

Comment:
How about `createRMQSinkWithPublishOptions()`? I am not a AMQP specialist, if `RMQSinkFeatured` has a special meaning, then leave it.

Reply:
I am using Featured but perhaps Enhanced should be a better choice. What do you think ?

Reply:
I see that you have already changed it to `createRMQSinkWithOptions`. I think this works.